lib/TPP/Dialect/Transform/LinalgXTransformOps.cpp

```diff
@@ -49,22 +49,15 @@ transform::PackOpExt::applyToOne(linalg::LinalgOp target,
         packedOp = mlir::linalgx::packConv2DNhwcHwcfOp(rewriter, convOp,
                                                        blockingFactors);
       })
+      .Case([&](linalg::GenericOp matmulOp) {
```

Comment:
Why this changes on the transform?

Reply:
Earlier, there was a structured pack transform that used to operate on gemm, now the plan is to get it to work on linalg.generic op to spit out vnni.brgemm instead of matmul. Do you think this needs to be removed entirely?